plugin/pkg/shoot/validator/admission.go

```diff
@@ -47,6 +47,9 @@ import (
 const (
 	// PluginName is the name of this admission plugin.
 	PluginName = "ShootValidator"
+
+	useAsSeedAnnotation = "shoot.gardener.cloud/use-as-seed"
```

Comment:
Can you rather use https://github.com/gardener/gardener/blob/d2b1097a54781271f67c91c91f0f14ea6e969f94/pkg/apis/core/v1beta1/constants/types_constants.go#L234?

Reply:
I have done it initially so, but It looks very strange to me imported something versioned in this package.
If you are fine, I can change it again.

Reply:
I think we should clean this up eventually (the constants suggest to be versioned due the directory structure, but in fact we don't have (and don't plan to have) multiple versions of them.